packages/@orchidui-vue/src/Form/Input/OcInput.vue

```diff
@@ -131,8 +140,8 @@ const inputClasses = computed(() => [
 
           <input
             ref="inputRef"
+            v-model="formattedValue"
```

Comment:
computed value without get/set should be `value`
```suggestion
            :value="formattedValue"
```